src/Impostor.Api/Net/Manager/ICompatibilityManager.cs

```diff
@@ -18,37 +19,50 @@ public enum VersionCompareResult
             Unknown,
         }
 
+        /// <summary>
+        /// Gets the compatibility groups.
+        /// </summary>
+        public IEnumerable<CompatibilityGroup> CompatibilityGroups { get; }
+
         /// <summary>
         /// Check if a client can join the server according to the currently accepted game versions.
         /// </summary>
         /// <param name="clientVersion">The client version to check for.</param>
         /// <returns>
         /// Whether this version is supported by the server at the moment and if not, whether it is too old or too new.
         /// </returns>
-        public VersionCompareResult CanConnectToServer(int clientVersion);
+        public VersionCompareResult CanConnectToServer(GameVersion clientVersion);
 
         /// <summary>Check if a player can join an existing game.</summary>
         /// <param name="hostVersion">The client version of the host.</param>
-        /// <param name="playerVersion">The client version of the player that is joining.</param>
+        /// <param name="clientVersion">The client version of the player that is joining.</param>
         /// <returns>
         /// - GameJoinError.None if everything is OK
         /// - GameJoinError.ClientOutdated if the player runs a too old game version
-        /// - GameJoinError.ClientTooNew if the player runs a too new game version
+        /// - GameJoinError.ClientTooNew if the player runs a too new game version.
```

Comment:
consistency: the other items also don't end on a `.`, remove this one or introduce it to the rest too